eng/common/scripts/SemVer.ps1

```diff
@@ -157,17 +177,26 @@ class AzureEngSemanticVersion {
 
   static [AzureEngSemanticVersion[]] SortVersions([AzureEngSemanticVersion[]] $versions)
   {
-    return ($versions | Sort-Object -Property Major, Minor, Patch, PrereleaseLabel, PrereleaseNumber -Descending)
+    return ($versions | `
+            Sort-Object -Descending -Property `
+              Major, Minor, Patch, PrereleaseLabel, PrereleaseNumber, `
+              @{ Expression = { [int]$_.BuildNumber }; Descending = $true })
```

Comment:
Does this work if BuildNumber is null/empty?

Reply:
Yeah it gets zeroed out:
```
$ [int]$null
0
$ $foo = @{}
$ [int]$foo.a
0
```